src/test/resources/application.yml

```diff
@@ -20,4 +20,22 @@ camel:
   springboot:
     tracing: false
 
-output : mock:output
+de.muenchen.mobidam.integration:
+  baseUrl: https://...
+  interfaces:
+    parkRideData:
```

Comment:
Alles was hier parkRideData heißt, müsstest Du noch umbenennen in parkRideStaticData. In der nächsten UserStory machen wir nämlich parkRideDynamicData

Reply:
@sfi2022 In InterfaceMessageFactory gibt es noch die Methoden "parkRideDataStart" etc. Nachdem das wahrscheinlich übergreifend über alle Interfaces benutzt wird, würde ich die in "mobilithekMessageStart" etc umbenennen (Achtung dabei auch den RouteBuilder anpassen).